src/docs/output-targets/custom-elements.md

```diff
@@ -37,24 +37,34 @@ import { HelloWorld } from 'my-library/dist/components/hello-world';
 customElements.define('hello-world', HelloWorld);
 ```
 
-The output directory will also contain an `index.js` file which exports all of
-your components and their respective `defineCustomElement` helper functions,
-looking something like this:
+The output directory will also contain an `index.js` file which exports some helper methods by default. The contents of the file
+will look something like (the contents may look different if [`customElementsExportBehavior`](#customelementsexportbehavior) is specified):
 
-```tsx
+```js
 export { setAssetPath, setPlatformOptions } from '@stencil/core/internal/client';
-export { MyComponent, defineCustomElement as defineCustomElementMyComponent } from './my-component.js';
-export {
-  MyOtherComponent,
-  defineCustomElement as defineCustomElementMyOtherComponent
-} from './my-other-component.js';
 ```
 
-This file can be used as the root module when distributing your component
-library, see [below](#distributing-custom-elements) for more details.
-
 ## Config
 
+### customElementsExportBehavior
+
+This config option provides additional behaviors that will alter the default component export OR custom element definition behaviors
+for this target. The desired behavior can be set via the following in a project's Stencil config:
+
+```ts
+{
+  type: 'dist-custom-elements',
+  customElementsExportBehavior: 'default' | 'single-export-module'
+}
+```
```

Comment:
Consider adding some additional context to this example to make it a little clearer where this configuration belongs. Some Stencil configs can get rather large (e.g. Ionic Framework Core). Providing some context helps frame the configuration options a bit more:
```suggestion
```ts
// stencil.config.ts
import { Config } from '@stencil/core';

export const config: Config = {
  outputTargets: [
    {
      type: 'dist-custom-elements',
      customElementsExportBehavior: 'default' | 'single-export-module'
    },
    // ...
  ],
// ...
}
```